lib/minesweeper_2pl/cat_emoji_formatter.rb

```diff
@@ -0,0 +1,114 @@
+module Minesweeper
+  class CatEmojiFormatter
```

Comment:
You are pretty close with this, but I think level of abstraction is slightly off. With `CatEmojiFormatter` you are duplicating a lot of logic from `BoardFormatter` which you want to avoid. Really the only thing different between the two classes is the actual emojis that are being used. You could keep a single board formatter, and move the duck type to something like `StandardEmojis` and `CatEmojis` that get passed into the `BoardFormatter`. We can work on this Wednesday if you want.